app/views/provider_interface/decisions/respond.html.erb

```diff
@@ -20,8 +20,13 @@
       <%- offer_text = 'Make an offer' %>
       <%- reject_text = 'Reject application' %>
       <%= f.govuk_radio_buttons_fieldset :decision, legend: { size: 'm', text: 'Choose response' } do %>
-        <%= f.govuk_radio_button :decision, 'offer', label: { text: offer_text }, link_errors: true %>
-        <%= f.govuk_radio_button :decision, 'reject', label: { text: reject_text } %>
+        <%= f.govuk_radio_button :decision, 'new_offer', label: { text: offer_text }, link_errors: true %>
+        <% if FeatureFlag.active?('provider_change_response') -%>
+          <%= f.govuk_radio_button :decision, 'edit_course', label: { text: "#{offer_text} but change course" }, link_errors: true %>
+          <%= f.govuk_radio_button :decision, 'edit_course_option', label: { text: "#{offer_text} but change location" }, link_errors: true %>
+          <%= f.govuk_radio_button :decision, 'edit_provider', label: { text: "#{offer_text} but change training provider" }, link_errors: true %>
```

Comment:
Should this option appear if the user isn't able to pick a different provider? 

![Screenshot 2020-03-24 at 11 30 38](https://user-images.githubusercontent.com/642279/77420967-ea49a780-6dc2-11ea-8209-076a62424efd.png)

Reply:
The change link on the previous page should not be visible if the user only belongs to one provider.